### ISSUE TITLE
Logs on 3.0.x no longer being ignored

### ISSUE THREAD

**Issue by poblahblahblah** [SumoLogic/sumologic-kubernetes-collection#2851]
**Describe the bug** A clear and concise description of what the bug is.

On the 2.x branch we were able to ignore all logs by default by setting the following value in our helm values.yaml file:

`sumologic.logs.container.excludeNamespaceRegex: '.*'`

We would then have services, deployments, etc. explicitly opt into log collection by adding the following annotations:

```
sumologic.com/include: "true"
```

This no longer appears to work, even though the documentation says it should.

Also, we tried to set `otelcolInstrumentation.sourceMetadata.excludeNamespaceRegex` to `'.*'` as well, but this also had no impact.

**Logs** Logs which can be helpful in investigating the issue.

N/A

**Command used to install/upgrade Collection** Direct command which you used to install/upgrade collection.

ArgoCD upgraded the chart.

**Configuration** Configuration used for Collection e.g. user-values.yaml for helm.

https://gist.github.com/poblahblahblah/952bf52aa4fe6ea4d6a422a122127f7d

**To Reproduce** Steps to reproduce the behavior.

Try to ignore logs, it should fail

**Expected behavior** A clear and concise description of what you expected to happen.

_most_ logs on our cluster should have been ignored.

**Environment (please complete the following information):**

- **Collection version** (e.g. `helm ls -n sumologic`): 3.0.0
- **Kubernetes version** (e.g. `kubectl version`): 1.23
- **Cloud provider**: AWS/EKS
- **Others**: N/A

**Anything else do we need to know** Add any other context about the problem here, e.g. issue happens only occasionally.

We're currently flooding Sumologic with nginx access logs.

**Comment by astencel-sumo** [comment]
Here is the workaround for all the regex exclusions: container, namespace, node and pod:

```yaml
metadata:
  logs:
    config:
      merge:
        processors:
          source/containers:
            exclude:
              container: <container-name-regex>
              namespace: <namespace-name-regex>
              node: <node-name-regex>
              pod: <pod-name-regex>
```

**Comment by astencel-sumo** [comment]
We have released chart [v3.0.1](https://github.com/SumoLogic/sumologic-kubernetes-collection/releases/tag/v3.0.1), please check it out.
